File: modules/workshop-charts/ios/WorkshopChartsModule.swift
import ExpoModulesCore

public class WorkshopChartsModule: Module {
  public func definition() -> ModuleDefinition {
    Name("WorkshopCharts")

    View(WorkshopChartsView.self) {
      Prop("data") { (view: WorkshopChartsView, prop: [Series]) in
        view.setChartData(data: prop)
      }
    }
  }
}

File: modules/workshop-charts/android/src/main/java/expo/modules/workshopcharts/WorkshopChartsModule.kt
package expo.modules.workshopcharts

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class WorkshopChartsModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("WorkshopCharts")

    View(WorkshopChartsView::class) {
      Prop("data") { view: WorkshopChartsView, prop: ArrayList<Series> ->
        view.setChartData(prop);
      }
    }
  }
}
